lib/lotus/view/configuration.rb

```diff
@@ -388,6 +409,7 @@ def reset!
         root             DEFAULT_ROOT
         default_encoding DEFAULT_ENCODING
 
+        @partials   = Hash.new
```

Comment:
@stevehook Not sure if it can work, but what about this?

```ruby
@partials = ::Hash.new {|h,k| h[k] = {} }
```

This should save us one line in `#add_partial`:

```ruby
def add_partial(key, format, partial)
  @partials[key][format] = partial
end
```

See http://ruby-doc.org/core-2.3.0/Hash.html#method-c-new